### PR TITLE
All countries map

### PR DESCRIPTION
**Pull request by SsbThatsMe**
This is a version of the map with much more detail than we previously showed, including 15 countries that were too small to be seen before. Some performance upgrades have been made in order to allow it to run (relatively) smoothly. It does have some stuttering when performing react updates (i.e. when zooming, selecting a country or changing the category) but I think that it could be made better over time with some performance updates.

### CODE REVIEW

**Comment by SsbThatsMe** [comment]
The hovered country does not show up in the legend right now since we're not triggering a react update. That might be fixed later.

**Comment by Willenbrink** [comment]
Looks good but I would like to have the markers back before merging this. I think the markers are more important than the resolution personally. Your call if you want to merge it for tonight.

**Review by marcusgahlin — APPROVED** [review]
Looks good! The increased level of detail on the map enables smaller countries such as Monaco to be found, which is very positive, but it makes the map less smooth performance vice. I guess we'll have to work with this.

The legend marker for the country that you're hoovering is also gone, we have to get that one back.

**Pull request merged**
